phpstan.neon

```diff
@@ -1,7 +1,10 @@
+includes:
+    - ./vendor/phpstan/phpstan-strict-rules/rules.neon
```

Comment:
I would put this (and the necessary changes) in a separate PR.

I think it makes sense to fix the stuff PHPStan detects with the standard checks first. And then consider enabling this. (When doing both at the same time, you can't tell if the strange Travis error is related to this or not)

Reply:
the changes are relatively minimal for it to pass the strict rules ?

Reply:
They probably are, so maybe it would be a wasted afford in this case.

Anyway, I just tried to build the `4.x` branch itself and it fails as well https://travis-ci.org/mhujer/uuid/jobs/341774850 - so it is probably caused by a new version of https://github.com/goaop/framework/releases

Reply:
https://github.com/ramsey/uuid/pull/213

Reply:
I've been giving some thought to separate pull requests on turning the error suppression back off- tinkering with an autoload-dev shim that matches the method signatures of the missing functions & constants, but throws `BadFunctionCallException` if inadvertently called ?